stories/DETableRow.stories.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from "react";
+
+import DETableRow from "../src/util/table/DETableRow";
+
+import { Table, TableBody, TableCell } from "@material-ui/core";
+import { storiesOf } from "@storybook/react";
+import DECheckbox from "../src/util/table/DECheckbox";
+
+export default class DETableRowTest extends Component {
+    render() {
+        return (
+            <Table>
+                <TableBody>
+                    <DETableRow hover>
+                        <TableCell>
+                            <DECheckbox checked={false} />
+                        </TableCell>
+                        <TableCell>Cell1</TableCell>
+                        <TableCell>Cell2</TableCell>
+                        <TableCell>Cell3</TableCell>
+                    </DETableRow>
+                    <DETableRow hover selected={true}>
+                        <TableCell>
+                            <DECheckbox checked={true} />
```

Comment:
Maybe one of these `DECheckbox` tests should have some custom children?

Reply:
I just realized - can/would you ever have a child component to a Checkbox?  I can't think of when you would 🤔

Reply:
Heh, good question. I'm not sure.

I thought maybe for the checkbox's text or label, but that's probably done through separate label and form control components 🤔 

I guess we could just remove `children` from `DECheckbox`, since this is meant for table rows, where I don't see the need for any `children` there.